#11 - Fix less imports for flow

andymikulski opened the pull request:
Fixes #11 

- Adds `NullModule` file needed for flow
- Corrects .flowconfig around the `less` imports

magopian commented on .flowconfig:
```diff
@@ -9,6 +9,7 @@
 [options]
+module.name_mapper.extension='less' -> '<PROJECT_ROOT>/NullModule.js'
 module.name_mapper='^console\/\(.*\)$' -> '<PROJECT_ROOT>/console/\1'
-module.name_mapper='.*\(.less\)$' -> 'empty/object'
```
I believe this would have worked if we had a dependency on https://www.npmjs.com/package/empty

andymikulski replied:
Yeah, I tried that after some searching, but it didn't work. Flow has been a real pain around "things that should work but don't, and no warnings or errors are thrown."

andymikulski commented:
I've changed the name to `FlowStubModule` (so it's easy to guess what it is for at first glance), and added a clarifying comment to the top of the file.

magopian commented on FlowStubModule.js:
```diff
@@ -0,0 +1,6 @@
+ * This is a stub file used by the Flow checker for filetypes we wish to ignore.
+ * Essentially, this tells Flow that the module does not export anything, so
+ * there is nothing for it to type check.
+ */
```
💯
